LICENSE.md

```diff
@@ -0,0 +1,6 @@
+Copyright 2019 - present
```

Comment:
```suggestion
Copyright 2019 - 2021 _ENTITY_
```
* `present` is not valid in a copyright statement; specific year(s) must be listed
* the entity claiming copyright must be named

Reply:
Re entity URI, let's revisit this after resolving https://github.com/solid/specification/pull/412 and if still applicable.